conf.yaml

```diff
@@ -1152,6 +1054,105 @@ contents:
                   path: /docs/kr
                   prefix: elasticsearch-extra/x-pack-elasticsearch
 
+    -
+        title:      Legacy Documentation
+        sections:
+          -
+            title:      X-Pack Reference
+            prefix:     en/x-pack
```

Comment:
Do we want to add versions to the title like we did for Marvel, etc.? "X-Pack Reference for 6.0-6.2 and 5.x"